Localization/cubature_kalman_filter/cubature_kalman_filter.py

```diff
@@ -0,0 +1,234 @@
+"""
+implementation of cubature kalman filter using CTRV motion model
+
+# author: Raghuram Shankar
+# based on PythonRobotics by Atsushi Sakai
+
+"""
+
+import math
+import matplotlib.pyplot as plt
+import numpy as np
+from scipy.linalg import sqrtm
+
+# initalize global variables
+dt = 0.1                                                # seconds
+# N = int(input('Enter number of samples : '))          # number of samples
```

Comment:
Please remove comment if it is not necessary.

Reply:
Sure, I will remove them.